src/main/java/duke/FileManager.java

```diff
@@ -0,0 +1,112 @@
+package duke;
+
+import duke.Tasks.DeadLineTask;
+import duke.Tasks.EventTask;
+import duke.Tasks.Task;
+import duke.Tasks.TodoTask;
+import duke.TextStoreAndPrint.TextPrinter;
+import duke.TextStoreAndPrint.TextStore;
+
+import java.io.*;
```

Comment:
Imported classes should be listed explicitly. Maybe you can change the IntelliJ setting, to only output * if more than 100 from same import, as I know IntelliJ will do this at times even if it is unchecked in the setting.

Reply:
noted with thanks